plugins/chat/app/models/chat/category_channel.rb

```diff
@@ -11,11 +11,13 @@ class CategoryChannel < Channel
       define_method(name) { true }
     end
 
+    # NOTE: keep in sync with `chat_allowed_groups`'s mandatory values
+    STAFF_GROUP_IDS ||= Group::AUTO_GROUPS.values_at(:admins, :moderators, :staff)
```

Comment:
```suggestion
    STAFF_GROUP_IDS = Group::AUTO_GROUPS.values_at(:admins, :moderators, :staff)
```

There is no longer a need to define constants with `||=` because the constant redefined warning we used to see is a relic of the past after we moved to zeitwerk for autoloading.

Reply:
Nice, TIL 👍